index.html

```diff
@@ -1,63 +1,82 @@
 <!DOCTYPE html>
 <html lang="en">
-<meta charset="utf-8">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="icon" type="image/png" href="data:image/png;base64,">
+<link rel="canonical" href="https://bestmotherfucking.website">
 <title>Best Motherfucking Website</title>
+<meta name="description" content="This website is fucking perfect. Seriously, what the fuck else do you want?">
+<meta property="og:type" content="website" />
+<meta property="og:title" content="Best Motherfucking Website" />
+<meta property="og:description" content="This website is fucking perfect. Seriously, what the fuck else do you want?" />
+<meta property="og:url" content="https://bestmotherfucking.website" />
+<meta property="og:site_name" content="Best Motherfucking Website" />
+<meta property="og:image" content="https://ipfs.io/ipfs/QmeVTyCq1JoN6Vhv5RQxWU28Lg6cQd3E273RwVwQpbX7Be?filename=og.png" />
+<meta property="og:image:width" content="1200" />
+<meta property="og:image:height" content="630" />
+<meta property="og:image:type" content="image/png" />
 <style>
-	@media (prefers-color-scheme: dark){
-		body {color:#fff;background:#000}
-		a:link {color:#cdf}
-		a:hover, a:visited:hover {color:#def}
-		a:visited {color:#dcf}
-	}
-	body{
-		margin:1em auto;
-		max-width:40em;
-		padding:0 .62em;
-		font:1.2em/1.62 sans-serif
-	}
-	h1,h2,h3 {
-		line-height:1.2
+	@media (prefers-color-scheme: dark) {
+		body { color:#fff;background:#000; }
+		a:link { color:#cdf; }
+		a:hover, a:visited:hover { color:#def; }
+		a:visited { color:#dcf; }
 	}
+	body{ margin:1em auto;max-width:40em;padding:0 .62em;font:1.2em/1.62 sans-serif;font-display:optional;font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif; }
```

Comment:
Please do use newlines for readability.

I'm also pretty sure leaving the fonts change out is preferred, see other issues and discussions.